cryptroot/nfpm.yaml

```diff
@@ -15,6 +15,8 @@ depends:
   - cryptsetup-bin
   - libgcc-s1
   - openssl
+  - xxd
+  - base64
```

Comment:
There is no base64 package; it's part of coreutils and unfortunately that's too large (~20MB)

Reply:
Done